.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+version: 2
+updates:
+  # Enable version updates for Gradle
+  - package-ecosystem: "gradle"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    target-branch: "main"
+    reviewers:
+      - "@DataBiosphere/broadworkspaces"
+      - "@rtitle"
```

Comment:
Not sure we want to bug Rob with these PRs, I think the `broadworkspaces` team should be ok.